src/metadatacache/MetadataCache.ts

```diff
@@ -338,19 +207,39 @@ export class MetadataCache {
     return `${this.url}`
   }
 
-  private transformResult(
-    { results, page, total_pages: totalPages, total_results: totalResults }: any = {
-      result: [],
-      page: 0,
-      total_pages: 0,
-      total_results: 0
-    }
-  ): QueryResult {
-    return {
-      results: (results || []).map((ddo: DDO) => new DDO(ddo as DDO)),
-      page,
-      totalPages,
-      totalResults
-    }
+  /**
+   * Simple blocking sleep function
+   */
+  public sleep(ms: number) {
```

Comment:
what's the purpose of this, how are users supposed to use this? If it's only for testing purposes it should not exist here but in some test helper file. Same for `waitForAqua()`

Reply:
we could use waitForAqua in the market as well, when publishing, instead of just doing a pause..